src/main/java/frc5124/robot2020/commands/auto/runpos/turretZero.java

```diff
@@ -0,0 +1,29 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc5124.robot2020.commands.auto.runpos;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc5124.robot2020.subsystems.Turret;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class turretZero extends InstantCommand {
```

Comment:
You can use a lambda to do this in an instant command instead of overriding initialize. This unnecessarily couples our command to the specifics of how an instant command works, instead of interfacing to the command and being agnostic of its internals. Alternatively, if lambdas are taboo, it would be better just to extend CommandBase.